bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.ts

```diff
@@ -179,7 +190,7 @@ export class KtbSequenceViewComponent implements OnInit, OnDestroy {
       )
       .subscribe(([params]: [Params, void]) => {
         if (params.shkeptncontext && this.project?.sequences) {
-          const sequence = this.project.sequences.find((s) => s.shkeptncontext === params.shkeptncontext);
+          const sequence = this.project?.sequences.find((s) => s.shkeptncontext === params.shkeptncontext);
```

Comment:
Optional chaining is not needed here because it's validated on top with `this.project?.sequences`